### ISSUE TITLE
Embedded subtitles take precedence over external subtitles

### ISSUE THREAD

**Issue by Deathspike** [jellyfin/jellyfin#6407]
**Describe the bug**

When a video file has an embedded English subtitle, as well as an external English subtitle, the embedded subtitle will be automatically selected as default subtitle.

**System:**
 - OS: Windows
 - Virtualization: Docker
 - Clients: Jellyfin Media Player & Web
 - Browser: Edge 92
 - Jellyfin Version: 10.7.6
 - Playback: ?
 - Hardware Acceleration: None
 - Installed Plugins: None
 - Reverse Proxy: None
 - Base URL: None
 - Networking: Bridge
 - Storage: local

**To Reproduce**
1. Add a `.MKV` file with embedded English subtitle, for example, `Slimes 01.mkv`.
2. Add an `.ass` English subtitle file, for example, `Slimes 01.en.ass`.
3. Scan the file with *Jellyfin*.
4. Click `Play` on the episode. The embedded subtitles will be used.

**Expected behavior**

As an end-user, I expect that external subtitles take precedence over embedded subtitles. I took the time to add these external subtitles, so I would expect a player to respect that and use them by default. This is what happens in other software I use, such as *MPC*, *Plex Media Player* and *VLC*. Selecting the embedded subtitle also causes unnecessary `ffmpeg` subtitle extraction and delays.

**Screenshots**
![image](https://user-images.githubusercontent.com/198601/128670207-0ae9bd36-1a23-450b-a0e1-597ac1f2ab88.png)

**Comment by TimGels** [comment]
I have to agree that it would be more consistent for end users if external subtitles took priority over embedded subtitle tracks.
